src/containers/BooksList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Book from './../components/Book';
+
+export const BooksList = () => {
+
+  return (
+    <table>
```

Comment:
- [ ] It should fetch the books from the redux store as mentioned here
<img width="468" alt="image" src="https://user-images.githubusercontent.com/86371332/201488860-c5c424fb-1180-4e30-a93f-3a755e5bb4bf.png">

Kindly use the [useSelector hook](https://react-redux.js.org/api/hooks#useselector) for this e.g. `const books = useSelector((state) => state.booksReducer);`